bitnami/apisix/templates/control-plane/dep-ds.yaml

```
@@ -0,0 +1,204 @@
+{{- if .Values.controlPlane.enabled }}
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: {{ ternary "DaemonSet" "Deployment" .Values.controlPlane.useDaemonSet }}
+metadata:
+  name: {{ template "apisix.control-plane.fullname" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/part-of: apisix
+    app.kubernetes.io/component: control-plane
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  {{- if and (not .Values.controlPlane.useDaemonSet) (not .Values.controlPlane.autoscaling.hpa.enabled) }}
+  replicas: {{ .Values.controlPlane.replicaCount }}
+  {{- end }}
+  {{- if .Values.controlPlane.updateStrategy }}
+  {{ ternary "updateStrategy" "strategy" .Values.controlPlane.useDaemonSet }}: {{- toYaml .Values.controlPlane.updateStrategy | nindent 4 }}
+  {{- end }}
+  selector:
+    matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
+      app.kubernetes.io/part-of: apisix
+      app.kubernetes.io/component: control-plane
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/control-plane/configmap.yaml") . | sha256sum }}
+        {{- if .Values.controlPlane.podAnnotations }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.controlPlane.podAnnotations "context" $) | nindent 8 }}
+        {{- end }}
+        {{- if .Values.commonAnnotations }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.commonAnnotations "context" $) | nindent 8 }}
+        {{- end }}
+      labels: {{- include "common.labels.standard" . | nindent 8 }}
+        app.kubernetes.io/part-of: apisix
+        app.kubernetes.io/component: control-plane
+        {{- if .Values.controlPlane.podLabels }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.controlPlane.podLabels "context" $) | nindent 8 }}
+        {{- end }}
+        {{- if .Values.commonLabels }}
+        {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 8 }}
+        {{- end }}
+    spec:
+      serviceAccountName: {{ template "apisix.control-plane.serviceAccountName" . }}
+      {{- if .Values.dataPlane.hostNetwork }}
+      hostNetwork: {{ .Values.dataPlane.hostNetwork }}
+      {{- end }}
```

I think the `dataPlane` reference here may be a typo
```suggestion
      {{- if .Values.controlPlane.hostNetwork }}
      hostNetwork: {{ .Values.controlPlane.hostNetwork }}
      {{- end }}
```